docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -4,6 +4,13 @@
 
 The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
 
+You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear under CUSTOM at the bottom of the page:
```

Comment:
All-caps CUSTOM feels shouty -- I know our [style guide](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#general-guidelines-for-writing-about-the-ui) says to reproduce the capitalization from the UI, but I think that's for Title Case or Sentence case, not all-caps. Maybe change to "Custom" (and bold?), or consider omitting entirely -- we may not need to name the section, just say where it is:
```suggestion
You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:
```